schema/timestamptz.go

```diff
@@ -35,6 +35,10 @@ func (*Timestamptz) Type() ValueType {
 	return TypeTimestamp
 }
 
+func (dst *Timestamptz) Size() int {
+	return 32
```

Comment:
How is this calculated? Because `time.Time` seems to be 24 bytes:
https://go.dev/play/p/K7awGEnpngi.

Reply:
Maybe infinity modifier has to be included, too?